samples/UsePolicy/Startup.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using Microsoft.AspNet.Builder;
+using Microsoft.AspNet.Cors.Core;
+using Microsoft.AspNet.Http;
+using Microsoft.Framework.DependencyInjection;
+
+namespace UsePolicy
+{
+    public class Startup
+    {
+        public void ConfigureServices(IServiceCollection services)
+        {
+            services.AddCors();
+        }
+
+        public void Configure(IApplicationBuilder app)
+        {
+            var allowSingleOrigin = new CorsPolicy();
+            allowSingleOrigin.Origins.Add("http://example.com");
+            app.UseCors(allowSingleOrigin);
```

Comment:
we chose to stick with 
```app.UseCors(builder => builder.AddOrigins("http://example.com"));```
and remove 
the one which takes in the policy object directly.